pcsx2/gui/AppMain.cpp

```diff
@@ -547,18 +570,35 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
-		if (EnableFMV == 1) {
+		if (EnableFMV) {
 			Console.Warning("FMV on");
 			DoFmvSwitch();
-			EnableFMV = 0;
+			EnableFMV = false;
 		}
 
 		if (FMVstarted){
 			int diff = cpuRegs.cycle - eecount_on_last_vdec;
 			if (diff > 60000000 ) {
```

Comment:
I was wondering the same actually... Maybe it's possible that all FMV's consume the specific cycle interval of the EE. Not sure though... I've never heard cases of random crashing though.